jbpm-wb-executor-service/jbpm-wb-executor-service-api/src/main/java/org/jbpm/workbench/es/model/events/ExecErrorSelectionEvent.java

```diff
@@ -16,22 +16,37 @@
 package org.jbpm.workbench.es.model.events;
 
 import org.jboss.errai.common.client.api.annotations.Portable;
+import org.jbpm.workbench.es.model.ExecutionErrorSummary;
 
 @Portable
 public class ExecErrorSelectionEvent {
```

Comment:
@nmirasch now that you're making changes here, would ou mind changing this class name to `ExecutionErrorSelectedEvent`  ? These events can also be moved into the client module as they are only used in there. 
Another note in the relation to error events, `ExecErrorSearchEvent` isn't in use, could be removed. `ExecErrorChangedEvent` is only sued in the presenter to refresh the grid, I would recommend deleting it too.